images/kube-proxy/Dockerfile.rhel

```diff
@@ -1,9 +1,17 @@
-FROM registry.svc.ci.openshift.org/ocp/4.0:hyperkube
+FROM registry.svc.ci.openshift.org/ocp/4.2:hyperkube
 
-RUN INSTALL_PKGS="iptables conntrack-tools" && \
+RUN INSTALL_PKGS="conntrack-tools" && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
     yum clean all && rm -rf /var/cache/*
 
+COPY ./images/sdn/scripts/iptables /usr/sbin/
```

Comment:
@squeed for my own knowledge (if you don't mind), what is the rational of having this iptables scripts?

Reply:
We don't know what mode the host's iptables is in - is it nftables compat mode, or legacy iptables mode? So, we just escape via chroot to the host and execute iptables there.